be/src/vec/exec/format/parquet/level_decoder.cpp

```diff
@@ -63,6 +63,20 @@ doris::Status doris::vectorized::LevelDecoder::init(doris::Slice* slice,
     return Status::OK();
 }
 
+doris::Status doris::vectorized::LevelDecoder::init_v2(doris::Slice* levels,
+                                                       doris::vectorized::level_t max_level,
+                                                       uint32_t num_levels) {
+    _encoding = tparquet::Encoding::RLE;
+    _bit_width = BitUtil::log2(max_level + 1);
+    _max_level = max_level;
+    _num_levels = num_levels;
+    size_t byte_length = levels->size;
+    _rle_decoder = RleDecoder<level_t>((uint8_t*)levels->data, byte_length, _bit_width);
+    levels->data += byte_length;
+    levels->size -= byte_length;
```

Comment:
Parameter `levels` will not be used anymore after this function, so we no need to rewrite the data and size.